src/js/utilities/characterSort.js

```diff
@@ -11,8 +11,8 @@ function getCharactersSortedByName(characters) {
 }
 
 function getSortedCharactersByPowerStat(characters, powerstat) {
-  powerstat.toLowerCase();
-  return characters.sort((a, b) => a.powerstats[powerstat] - b.powerstats[powerstat]);
+  // eslint-disable-next-line max-len
+  return characters.sort((a, b) => a.powerstats[powerstat.toLowerCase()] - b.powerstats[powerstat.toLowerCase()]);
```

Comment:
Ok. Jordan mentioned this too, so I can go back and fix those. It might take me a bit- I know Jared cleaned up our `characterSort.spec` file, so now the test for powerstats fails!! 😣

Reply:
🏆